### DISCUSSION TITLE
Open Images Object Detection

### DISCUSSION THREAD

**Discussion by dannycho1530** [100-hours-a-week/discussions#72] [Deepdive]
https://www.kaggle.com/c/open-images-object-detection-rvc-2020

Open Images Object Detection

Open Images 데이터셋을 사용하여 다양한 객체를 탐지하는 프로젝트
다양한 객체 클래스를 탐지하고, 복잡한 이미지에서 다수의 객체를 정확하게 식별하는 것
Faster R-CNN, RetinaNet, YOLO, 데이터 증강, Transfer Learning

프로젝트 진행에 사용될 기법 (하단의 기법 중 1택 아니면 2택하여 심도있게 딥다이브):
1. 데이터 전처리 및 증강
2. 모델 구조 및 성능 향상
3. 하이퍼파라미터 최적화
4. 실시간 랜드마크 검출

**Comment by ej31** [comment]
* 메모 : @dannycho1530 는 AI 회사를 지원할 예정이기 때문에 팀프로젝트와 연관없는 별도의 주제로 딥다이브 진행
